apps/site/assets/css/_events.scss

```diff
@@ -40,3 +42,30 @@
 .event-inline-block {
   display: inline-block;
 }
+
+.m-event__date-circle {
```

Comment:
If you added the `font-weight: 700` here then you wouldn't need to adjust the existing small CSS.

Reply:
Yeahh, that's part of why I was asking about how much we're supposed to match the ZeroHeight rules.

[The event listing anatomy](https://zeroheight.com/2fedee66c/p/9010eb-unique-components/i/29969884) has the text in the circle represented by <small>.  [And the typography definition of <small>](https://zeroheight.com/2fedee66c/p/37413e-typography/b/83bfe5) has font-weight 700.  

The only other place "small" is used is on the "Contact Us" page.  This is what it looks like if <small> has font-weight 700.  Doesn't look quite right.  So should I change this to another non-small element, or should <small> be defined differently in our typography system?